datahub-web-react/src/app/entity/shared/containers/profile/header/PlatformContent/PlatformContentContainer.tsx

```diff
@@ -15,19 +15,18 @@ export function getDisplayedEntityType(
     entityRegistry: EntityRegistry,
     entityType: EntityType,
 ) {
-    const entityTypeCased =
-        (entityData?.subTypes?.typeNames?.length && capitalizeFirstLetterOnly(entityData?.subTypes.typeNames[0])) ||
-        entityRegistry.getEntityName(entityType);
-    return entityData?.entityTypeOverride || entityTypeCased || '';
+    return (
+        entityData?.entityTypeOverride ||
+        capitalizeFirstLetterOnly(entityData?.subTypes?.typeNames?.[0]) ||
```

Comment:
is it safe to remove the check entityData?.subTypes?.typeNames?.length

What if type names does not exist?

Reply:
(I see that the method can deal with undefined), but I think the other case is when the array exists, but is empty (technically a possible state)